[공통 컴포넌트] RoundBox 컴포넌트 추가

Ho2eny opened the pull request:
### 요구 사항
![image](https://user-images.githubusercontent.com/50590025/185778594-61543f6e-ab19-482b-90aa-971420420070.png)

### 당부의 말씀
- `width: 100%, height: 100%`로 부모의 크기에 맞췄습니다

### 테스트 결과
![image](https://user-images.githubusercontent.com/50590025/185778642-6ea5d251-69a2-4e36-b011-74bed10dacea.png)
![image](https://user-images.githubusercontent.com/50590025/185778674-f2bb1ba0-6cd4-4c3a-a0c1-44f8a990a430.png)

Ho2eny committed:
feat: RoundBox 컴포넌트를 추가합니다

wlsdhr0831 commented on src/components/RoundBox/RoundBox.scss:
```diff
@@ -0,0 +1,12 @@
+.round-box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
해당 값은 필요 없을 것 같습니다~
내부 디자인에 대한 css가 flex가 필수가 아니고,
이미 적용되어있는 기본 값은 컴포넌트 재사용시 불편함을 초래하지 않을까 싶네요~!

Ho2eny replied:
네, 동의합니다. 수정했습니다

wlsdhr0831 commented on src/components/RoundBox/RoundBox.tsx:
```diff
@@ -0,0 +1,20 @@
+import classNames from "classnames";
+import "./RoundBox.scss";
+
+export interface Props {
+  hasShadow: boolean;
+  children?: React.ReactNode;
+}
+
+const RoundBox = ({
+  children,
+  hasShadow
+}: Props): React.ReactElement => {
```
hasShadow를 옵셔날한 값으로 변경하고
11번째 줄에 더 많이 적용되어있는 디자인을 디폴트 값으로 입력해두어도 좋을것같습니다!

디폴트값 적용은 11번째 줄에
`hasShadow = false` 이렇게 작성하시면 됩니다!

Ho2eny replied:
좋은 의견 감사합니다!! 👍

wlsdhr0831 reviewed (approved):
👍 👍

Ho2eny committed:
fix: 리뷰를 반영합니다

- display, felx 등 children의 스타일을 지우고 classNames를 추가합니다
- hasShadow에 default 값을 주고 Optional하게 변경합니다
